Prog/Updates/Update_MFIS_2022_12.sas

```diff
@@ -0,0 +1,26 @@
+/**************************************************************************
+ Program:  Update_MFIS_2022_12.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Donovan Harvey
+ Created:  1/25/23
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ GitHub issue:  Update-#332
+ 
+ Description:  Update Preservation Catalog with latest 
+ HUD MFIS update file.
+
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( HUD )
+
+
+%Update_MFIS( Update_file=MFIS_2022_12 )
```

Comment:
As you noted, there are a few loans here that are not matching up with their respective projects. That is because the subsidy record has not been created for these loans. I will create a separate issue for this update, which we should complete before finalizing this issue.